api-reference/50 Common/Object Structures/positionConfig/positionConfig.md

```diff
@@ -4,11 +4,99 @@ type: Object
 ---
 ---
 ##### shortDescription
-The **position** object specifies the widget positioning options.
+Configures the position of an overlay element.
 
 ---
-This object is passed to the **position** configuration option of a widget that overlays the main screen ([LoadPanel](/api-reference/10%20UI%20Widgets/dxLoadPanel '/Documentation/ApiReference/UI_Widgets/dxLoadPanel/'), [Popup](/api-reference/10%20UI%20Widgets/dxPopup '/Documentation/ApiReference/UI_Widgets/dxPopup/'), [Popover](/api-reference/10%20UI%20Widgets/dxPopover '/Documentation/ApiReference/UI_Widgets/dxPopover/'), and [Toast](/api-reference/10%20UI%20Widgets/dxToast '/Documentation/ApiReference/UI_Widgets/dxToast/')).
+This object should be assigned to the **position** option of an overlay widget ([Popup](/Documentation/ApiReference/UI_Widgets/dxPopup/), [Popover](/Documentation/ApiReference/UI_Widgets/dxPopover/), [Tooltip](/Documentation/ApiReference/UI_Widgets/dxTooltip/), etc.).
 
-The positionConfig object may contain the following fields: **my**, **at**, **of**, **offset** and **collision**. Look at the following sentence to see how to use these fields to position the required element against the target element.
+To position an element, specify the [my](/Documentation/ApiReference/Common/Object_Structures/positionConfig/my/), [at](/Documentation/ApiReference/Common/Object_Structures/positionConfig/at/), and [of](/Documentation/ApiReference/Common/Object_Structures/positionConfig/of/) options. In the following code, the left side of the **Popup** widget is aligned with the right side of a target. This configuration reads as follows: "place **my** *left* side **at** the *right* side of the *#target* element."
```

Comment:
Change the second sentence to:
In the following code, the **Popup** widget's left side is aligned with the target's right side.